src/site/_includes/shows-individual.njk

```diff
@@ -1,39 +1,41 @@
 ---
-layout: default
+layout: default-next
 ---
-{# *-landing-page classes are a holdover from devsite. #}
-{# It would be great to remove them as part of the v1 migration. #}
-{% from 'macros/show-card.njk' import showCard with context %}
-<div class="shows-landing-page">
-  {# Make sure Actions are first in the tab order. #}
-  <div class="w-actions">
+<div class="authors-individual">
+  {# @TODO fix subscribe button #}
+  {# <div class="w-actions">
     {% include 'partials/subscribe-action.njk' %}
-  </div>
+  </div> #}
 
-  <div class="w-layout-container w-pb--non">
-    <header class="w-page-header">
-      <h1 class="w-page-header__headline w-tags-page-header__headline {{'w-visually-hidden' if paged.hero.heroOnly}}">{{ renderData.title }}</h1>
-      <p class="w-page-header__copy">{{ renderData.description | safe }}</p>
-      <a href="{{ renderData.rss }}" target="_blank" class="w-tags-page__rss">
-        <img src="/images/icons/rss.svg" alt="RSS Feed" width="32" height="32">
-      </a>
-    </header>
-  </div>
-  <section class="w-grid">
-    <div class="w-grid__columns w-grid__columns--three" role="list">
-      {% for show in paged.elements %}
-        {{ showCard(show) }}
-      {% endfor %}
+  {# Hero starts #}
+  <header class="region bg-mid-bg">
+    <div class="wrapper">
+      <div class="headline all-center flow">
+        <h1 class="headline__title">{{ renderData.title }}</h1>
+        <p>{{ renderData.description | safe }}</p>
+        <div class="cluster all-center">
+          <div>
+            <a href="{{ renderData.rss }}" target="_blank">
+              <img src="/images/icons/rss.svg" alt="RSS Feed" width="32" height="32">
+            </a>
+          </div>
+        </div>
+      </div>
     </div>
-  </section>
+  </header>
+  {# Hero ends #}
 
-  {% include 'pagination.njk' %}
+  {% include 'partials/paged-next.njk' %}
 
-  <div class="w-layout-container">
-    <div class="w-text--center">
-      <a class="w-button" href="/shows">
-        {{ 'i18n.paths.shows.see_more' | i18n(locale) }}
-      </a>
+  <div class="region">
+    <div class="wrapper">
+      <div class="cluster all-center">
+        <div>
+          <a class="button" href="/shows">
+            {{ 'i18n.paths.shows.see_more' | i18n(locale) }}
```

Comment:
This string is repeated in tags and author pages. Please move to i18n.common.see_more instead and reuse.

Reply:
They're specific, 'see more shows', 'see more authors', 'see more tags'. I don't think it can be broken up into 'see more' + 'authors' as some languages will handle that sentence structure differently.